docker-compose.yml

```diff
@@ -0,0 +1,16 @@
+version: "3.8"
+services:
+  postgres:
+    image: kartoza/postgis:12.0
```

Comment:
Is there any reason to use this one over the official `postgis/postgis` image?

Reply:
I mainly went with this one because it supports SSL out of the box, so I don't need to faff around setting it up manually or toggle the flag when we're not running in production. It also supports booting up multiple databases as part of the environment config which was nice